vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -302,14 +302,23 @@ def _make_page_layout(self, page: Page, **kwargs):
         page_layout.id = page.id
         return page_layout
 
-    @staticmethod
-    def _make_page_404_layout():
+    def _make_page_404_layout(self):
         # The svg file is available through the _dash-component-suites/vizro route, as used in Dash's
         # _relative_url_path, but that feels too private to access directly. Hence read the file in directly rather
         # than referring to its path.
         error_404_svg = base64.b64encode((VIZRO_ASSETS_PATH / "images/error_404.svg").read_bytes()).decode("utf-8")
         return html.Div(
             [
+                html.Div(
```

Comment:
I added the theme switch to the 404 page. It's kind of hidden and looks as part of the image - I kind of like it :D 

This at least gets rid of the theme flickering when switching from a light theme to the 404 page. The inconsistency in themes is still present for the loading page, but I guess that's what we always accepted.

Reply:
A comment here would be good about why we have a theme selector the 404 page.